src/Lean/Meta/ExprDefEq.lean

```diff
@@ -1263,10 +1263,7 @@ private def processAssignment' (mvarApp : Expr) (v : Expr) : MetaM Bool := do
 
 private def isDeltaCandidate? (t : Expr) : MetaM (Option ConstantInfo) := do
   match t.getAppFn with
-  | Expr.const c _ =>
-    match (← getUnfoldableConst? c) with
-    | r@(some info) => if info.hasValue then return r else return none
```

Comment:
`hasValue` checks unnecessary now that `getUnfoldableConst*?` is documented to return defs/thms.